manifests/global.yaml

```diff
@@ -40,7 +40,8 @@ global:
 
   # Default tag for Istio images.
   tag: latest
-
+  # clusterID for the cluster 
+  clusterID: ""
```

Comment:
We should probably mark the existing `global.multiCluster.clusterName` as deprecated? `global.clusterName` would be used in templates anywhere `global.multiCluster.clusterName` is. For example, in the ingress gateway deployment and sidecar injection templates we have:

```         - name: ISTIO_META_CLUSTER_ID
            value: "{{ $.Values.global.multiCluster.clusterName | default `Kubernetes` }}"
````

... which would change to:

```
         {{ $deprecatedClusterID := .Values.global.multiCluster.clusterName | default "Kubernetes" }}
          - name: ISTIO_META_CLUSTER_ID
            value: "{{ .Values.global.clusterID | default $deprecatedClusterID }}"
````

Reply:
Thanks! changed accordingly